common/include/fwd-event.h

```diff
@@ -15,6 +15,7 @@ struct d_change_event;
 struct d_select_event;
 
 enum event_type : unsigned;
+enum class window_event_result : unsigned;
```

Comment:
Aha, I was really confused about this error because I thought the types were defined in `sys/types.h`. Apparently not for C++. Fixed, including `cstdint` at the top of `pstypes.h`.